### PR TITLE
Add contentcreated date-time field

### PR DESCRIPTION
**Pull request by NiallBunting**
This will capture the date-time when the original asset was created
i.e. The time a photographer takes the photo according to the camera.

Added the following to the 1.2 schema:
```
+    "contentcreated": {
+      "description": "The date and time when the asset was created",
+      "type": "string",
+      "format": "date-time"
+    },
```

Implements: IAI-1316

### CODE REVIEW

**Review by iyoung — APPROVED** [review]
Looks good to me, one non-required new date-time field contentcreated added

File: content/ninjs/pa-ninjs-schema_1.3.json [hunk 1]
```diff
@@ -0,0 +1,425 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://schema.pressassociation.io/content/ninjs/pa-ninjs-schema_1.3.json#",
+  "type": "object",
+  "title": "PA IPTC ninjs - News in JSON - version 1.1 (Derived from IPTC - News in JSON - version 1.1)",
```
Comment (petersow) [inline content/ninjs/pa-ninjs-schema_1.3.json]: version 1.1 -> version 1.3 (I forgot to update for 1.2)

File: content/ninjs/pa-ninjs-schema_1.3.json [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+        }
+      }
+    },
+    "associations": {
+      "description": "Content of news objects which are associated with this news object.",
+      "type": "object",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^[a-zA-Z0-9]+": {
+          "$ref": "http://schema.pressassociation.io/content/ninjs/pa-ninjs-schema_1.1.json#"
```
Comment (petersow) [inline content/ninjs/pa-ninjs-schema_1.3.json]: bump to 1.3, to self refer

**Review by iyoung — APPROVED** [review]

**Comment by petersow** [comment]
Could we add a 1.3 version of the list please?

**Pull request merged**